templates/azure/ebook/index.html

```diff
@@ -10,100 +10,73 @@
+  <div class="u-fixed-width p-section--shallow">
+    <hr />
+    <h2>Ubuntu Pro for Azure is a premium image delivering the most comprehensive open source security and compliance</h2>
   </div>
-
-  <div class="p-strip">
-    <div class="row--50-50">{% include "azure/shared/_how-ubuntu-pro-helps-you.html" %}</div>
+  <div class="row--50-50">
+    <div class="col">
+      <p class="p-heading--5">70% of Azure virtual machines are running on Ubuntu due to reliability and ease of use</p>
+      <p>As the publishers of Ubuntu, we went one step further and developed a premium Ubuntu Pro image in partnership with public cloud providers, to deliver a preferred open-source solution for cloud innovators.</p>
+      <p>Here is our latest eBook, where we share the details of how Ubuntu Pro on Azure delivers extended open-source security, enterprise compliance, 10 years of extended maintenance and more.</p>
+    </div>
+    <div class="col" id="ubuntu-pro-ebook">
+      {% with id="4393", returnURL="/azure/ebook/thank-you", cta="Get the eBook", cta_class="p-button--positive" %}
+        {% include "engage/shared/_en_engage_form.html" %}
+      {% endwith %}
+    </div>
   </div>
+</section>
 
-  <div class="p-strip--light">
-    <div class="row">
-      <div class="col-7">
-        <h2>Level up your Ubuntu journey by activating this private Azure Marketplace offer</h2>
+<section class="p-section">
+    {% include "azure/shared/_how-ubuntu-pro-helps-you.html" %}
+</section>
 
-        <p>Already running Ubuntu on Microsoft Azure, or planning a migration to the public cloud?</p>
-
-        <p>Leverage Canonical's expert service and all the features of Ubuntu Pro with this exclusive private offer.</p>
-
-        <a href="/azure/private-offer" class="p-button--positive">Learn more</a>
-      </div>
-
-      <div class="col-5 u-align--center">
-        {{ image(url="https://assets.ubuntu.com/v1/71fe6a11-private-offer-shield.svg",
-                alt="",
-                width="206",
-                height="234",
-                hi_def=True,
-                loading="lazy") | safe
-        }}
+<section class="p-section--deep">
+  <div class="row--50-50">
+    <hr />
+    <div class="col">
+      <h2>Level up your Ubuntu journey by activating this private Azure Marketplace offer</h2>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow">
+        <div class="p-image-container--3-2 is-highlighted u-hide--small">
+          <img class="p-image-container__image p-strip is-shallow" style="width: 30%;"
+            src="https://assets.ubuntu.com/v1/71fe6a11-private-offer-shield.svg" alt="" />
+        </div>
       </div>
+      <p>Already running Ubuntu on Microsoft Azure, or planning a migration to the public cloud?</p>
+      <p>Leverage Canonical's expert service and all the features of Ubuntu Pro with this exclusive private offer.</p>
+      <hr class="is-muted" />
+      <a href="/azure/private-offer" class="p-button">Learn more</a>
```

Comment:
To make this CTA more descriptive
```suggestion
      <a href="/azure/private-offer" class="p-button">Learn more <span class="u-off-screen">about our Azure Marketplace private offer</span></a>
```